pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -4873,6 +4895,7 @@ TOKEN [IGNORE_CASE]:
 <UI: "UI"> |
 <UNDER: "UNDER"> |
 <USING: "USING"> |
+<WAIT: "WAIT"> |
```

Comment:
I tend to not declare these keywords as tokens anymore, since the list becomes really long - and soon, we'll hit the java compiler limit for switch statement cases when using ID() - each possible identifier, that can also be a keyword, would be listed then.

I use the technique via a lookahead and consume a specific identifier:

`LOOKAHEAD({isKeyword("WAIT")}) KEYWORD("WAIT")`